completion/jiwonlipsum.cwl

```diff
@@ -0,0 +1,18 @@
+# jiwonlipsum package
+# ponte-vecchio 2023-01-17 for v0.6
+
+#keyvals:\usepackage/jiwonlipsum#c
+nopar
+numbers
+#endkeyvals
+
+\jiwon
+\jiwon[%<num%>]
+\jiwon[%<char%>]
```

Comment:
do you really want to define the same command twice, only with different argument name ?

Reply:
Changed to `jiwon[%<num/char%>]`